pkg/plaid_cli/linker.go

```diff
@@ -289,56 +293,28 @@ var relinkTemplate string = `<html>
     <script type="text/javascript">
      (function($) {
        var handler = Plaid.create({
-         clientName: 'plaid-cli',
-         // Optional, specify an array of ISO-3166-1 alpha-2 country
-         // codes to initialize Link; European countries will have GDPR
-         // consent panel
-         countryCodes: ['US'],
-         env: '{{ .Environment }}',
-         // Replace with your public_key from the Dashboard
-         key: '{{ .PublicKey }}',
-         product: ['transactions'],
-         token: "{{ .PublicToken }}",
-         language: 'en',
-         onLoad: function() {
-           // Optional, called when Link loads
-         },
-         onSuccess: function(public_token, metadata) {
-           console.log("onSuccess");
-           // Send the public_token to your app server.
-           // The metadata object contains info about the institution the
-           // user selected and the account ID or IDs, if the
-           // Select Account view is enabled.
-           $.post('/link', {
-             public_token: public_token,
-           });
-         },
-         onExit: function(err, metadata) {
-           if (err != null) {
-             console.log(err);
-
-             // Handle manual relink when credential is already valid
-             if (err.error_code == "item-no-error") {
-               $.post('/link', {
-                 public_token: "{{ .PublicToken }}",
-               });
-             }
-           }
-
-           document.getElementById("alert").classList.remove("hidden");
-         },
-         onEvent: function(eventName, metadata) {
-           // Optionally capture Link flow events, streamed through
-           // this callback as your users connect an Item to Plaid.
-           // For example:
-           // eventName = "TRANSITION_VIEW"
-           // metadata  = {
-           //   link_session_id: "123-abc",
-           //   mfa_type:        "questions",
-           //   timestamp:       "2017-09-14T14:42:19.350Z",
-           //   view_name:       "MFA",
-           // }
-         }
+	 token: '{{ .LinkToken }}',
+	 onSuccess: (public_token, metadata) => {
```

Comment:
For some reason Plaid requires this `onSuccess` method to be defined, but doesn't seem to actually call it for a relink. Go figure...